br/pkg/restore/client_test.go

```diff
@@ -23,17 +23,15 @@ import (
 	"google.golang.org/grpc/keepalive"
 )
 
+var MC *mock.Cluster
```

Comment:
this does not need to be public.

call the variable `mockCluster` or something descriptive.